### ISSUE TITLE
Emptying trash does not empty it in the web app

### ISSUE THREAD

**Issue by rachelsquirrel** [Automattic/simplenote-android#340]
Empty the Trash in the Android app and then sign into the same account with the web app and view the Trash. Notes have disappeared from the Android app Trash but are still visible in the Trash in the web app.

Ticket 2511571

**Comment by Decadence** [comment]
Same for me

**Comment by geekpete** [comment]
I found that when signing out and signing back in again, the Trash was then empty.
It must perform a sync of Trash on login

So make this either a manual button or automatic as the rest of the syncing is. 
Unless there could be a setting to only make local trash empty a manual procedure in case you wanted to salvage accidentally deleted notes that might still remain on your phone? But in this case, just provide a note versioning or archival service that you can retrieve from that retains emptied trash items for 30 days on the server perhaps...

**Comment by alisterscott** [comment]
What happens if you delete the app and reinstall before logging back in?

**Comment by theck13** [comment]
I reproduced that behavior using a Google Pixel 3 on Android 9.0 with Simplenote 1.7.0.  I confirmed the following:
- Trash emptied on Android with notes deleted on Android removes all notes on Android and web.
- Trash emptied on web with notes deleted on Android removes all notes on Android and web.
- Trash emptied on Android with notes deleted on web removes all notes on Android, but not web.
- Trash emptied on web with notes deleted on web removes all notes on Android and web.

This _may_ be related to https://github.com/Automattic/simplenote-android/issues/279.
